repos/fdbt-admin/package.json

```diff
@@ -45,7 +48,7 @@
         "lint:fix": "eslint --fix . --ext .ts,.tsx",
         "test:ci": "CI=true react-scripts test",
         "preinstall": "npx npm-force-resolutions",
-        "precommit": "lint-staged && npm run test:ci && tsc"
+        "precommit": "npm run test:ci && tsc"
```

Comment:
are we removing the lint-staged

Reply:
Yeah - the babel transformation which is needed for the compilation to work isn't working with the linting, so I have disabled the linting from running on commit.